src/domain/synthetics/trade/useSelectedTradeOption.ts

```diff
@@ -33,6 +42,14 @@ export type SelectedTradeOption = {
   setMarketAddress: (marketAddress?: string) => void;
   setCollateralAddress: (tokenAddress?: string) => void;
   switchTokenAddresses: () => void;
+  setTradeOptions: ({
```

Comment:
trade option should be in the singular here i think, roughly speaking it's trade config